README.rst

```diff
@@ -24,17 +16,13 @@ Installation
 
 ``j2y`` is installed using ``pip``::
```

Comment:
i think it would be nice to add a link to the original source and thank @chaudum for the project!